crates/ruff_python_formatter/tests/snapshots/format@expression__fstring.py.snap

```diff
@@ -516,6 +562,10 @@ f"""test {"inner"}"""
 # But if the inner string is also triple-quoted then we should preserve the existing quotes.
 f"""test {'''inner'''}"""
 
+# Not valid Pre 3.12
+f"""test {f"inner {'''inner inner'''}"}"""
```

Comment:
It's interesting that as a side effect we'll start fixing certain code that would raise a syntax error pre-3.12.